mozetl/shield/containers.py

```diff
@@ -0,0 +1,34 @@
+from pyspark.sql.types import StringType, LongType
+
+from ..basic import convert_pings, DataFrameConfig
+from .utils import testpilot_etl_boilerplate
+
+
+def transform_pings(sqlContext, pings):
```

Comment:
I'm not sure how the plumbing of mozetl works ... can I simply put this at the bottom?
```
testpilot_etl_job = testpilot_etl_boilerplate(
      transform_testpilot_pings,
      's3n://telemetry-parquet/harter/containers_testpilottest/v1'
  )

shield_etl_job = testpilot_etl_boilerplate(
      transform_shield_pings,
      's3n://telemetry-parquet/harter/containers_shield/v1'
  )
```

Or can there be only 1 `etl_job` symbol in the module?